src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -365,15 +367,20 @@ public bool DiagnosticSourceEnabled
         /// <seealso cref="ConfigurationKeys.HttpClientErrorStatusCodes"/>
         internal bool[] HttpClientErrorStatusCodes { get; set; }
 
+        /// <summary>
+        /// Gets configuration values for changing service names based on configuration
+        /// </summary>
+        internal ServiceNames ServiceNameMappings { get; }
+
         /// <summary>
         /// Gets a value indicating the size of the trace buffer
```

Comment:
```suggestion
        /// Gets a value indicating the size of the trace buffer in bytes
```

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1151/commits/036ae23d5ace2c8e19264ac624a2c0ec63de27e6